spacy/cli/debug_data.py

```diff
@@ -437,110 +357,93 @@ def debug_data(
+                ", ".join(set(labels_dev) - set(labels_train)),
                 show=verbose,
             )
 
         if has_low_data_warning:
             msg.text(
-                "To train a parser, your data should include at "
-                "least {} instances of each label.".format(DEP_LABEL_THRESHOLD),
+                f"To train a parser, your data should include at "
+                f"least {DEP_LABEL_THRESHOLD} instances of each label.",
                 show=verbose,
             )
 
         # multiple root labels
         if len(gold_train_unpreprocessed_data["roots"]) > 1:
             msg.warn(
-                "Multiple root labels ({}) ".format(
-                    ", ".join(gold_train_unpreprocessed_data["roots"])
-                )
-                + "found in training data. spaCy's parser uses a single root "
-                "label ROOT so this distinction will not be available."
+                f"Multiple root labels "
+                f"({', '.join(gold_train_unpreprocessed_data['roots'])}) "
+                f"found in training data. spaCy's parser uses a single root "
+                f"label ROOT so this distinction will not be available."
             )
 
         # these should not happen, but just in case
         if gold_train_data["n_nonproj"] > 0:
             msg.fail(
-                "Found {} nonprojective projectivized train sentence{}".format(
-                    gold_train_data["n_nonproj"],
-                    "s" if gold_train_data["n_nonproj"] > 1 else "",
-                )
+                f"Found {gold_train_data['n_nonproj']} nonprojective "
+                f"projectivized train sentence(s)"
             )
         if gold_train_data["n_cycles"] > 0:
             msg.fail(
-                "Found {} projectivized train sentence{} with cycles".format(
-                    gold_train_data["n_cycles"],
-                    "s" if gold_train_data["n_cycles"] > 1 else "",
-                )
+                f"Found {gold_train_data['n_cycles']} projectivized train sentence(s) with cycles"
             )
 
     msg.divider("Summary")
     good_counts = msg.counts[MESSAGES.GOOD]
     warn_counts = msg.counts[MESSAGES.WARN]
     fail_counts = msg.counts[MESSAGES.FAIL]
     if good_counts:
-        msg.good(
-            "{} {} passed".format(
-                good_counts, "check" if good_counts == 1 else "checks"
-            )
-        )
+        msg.good(f"{good_counts} {'check' if good_counts == 1 else 'checks'} passed")
     if warn_counts:
-        msg.warn(
-            "{} {}".format(warn_counts, "warning" if warn_counts == 1 else "warnings")
-        )
-    if fail_counts:
-        msg.fail("{} {}".format(fail_counts, "error" if fail_counts == 1 else "errors"))
-
+        msg.warn(f"{warn_counts} {'warning' if warn_counts == 1 else 'warnings'}")
     if fail_counts:
+        msg.fail(f"{fail_counts} {'error' if fail_counts == 1 else 'errors'}")
         sys.exit(1)
```

Comment:
as in the cases you've corrected above, I guess these could also just simply be "check(s"), "warning(s)" and "error(s)"